client/blocks/term-tree-selector/terms.jsx

```diff
@@ -388,6 +388,17 @@ const TermTreeSelectorList = React.createClass( {
 		);
 	},
 
+	cellRendererWrapper: function( { key, style, ...rest } ) {
```

Comment:
We can (and should, at least for the purposes of consistency) use ES6 function value syntax here:

```js
cellRendererWrapper( { key, style, ...rest } ) {
```